# Thomas Hoffmann, EMBL Heidelberg, structures-it@embl.de, 2021/09
# change requirements in setup.py from tensorflow-cpu to tensorflow.
diff -ru alphafold-1d43aaff941c84dc56311076b58795797e49107b/setup.py alphafold-1d43aaff941c84dc56311076b58795797e49107b_setup_py_remove_tfcpu/setup.py
--- alphafold-1d43aaff941c84dc56311076b58795797e49107b/setup.py	2021-09-10 14:34:22.000000000 +0200
+++ alphafold-1d43aaff941c84dc56311076b58795797e49107b_setup_py_remove_tfcpu/setup.py	2021-09-20 11:13:44.817481819 +0200
@@ -39,7 +39,7 @@
         'ml-collections',
         'numpy',
         'scipy',
-        'tensorflow-cpu',
+        'tensorflow',
     ],
     tests_require=['mock'],
     classifiers=[
Only in alphafold-1d43aaff941c84dc56311076b58795797e49107b_setup_py_remove_tfcpu: .setup.py.swo
